images/dind/dind.conf

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Docker-in-Docker (dind)
+
+[Service]
+ExecStart=
```

Comment:
It is needed.  As per a previous comment, dind.conf is a drop-in unit file that overrides what appears in the distro docker.service.  In order to override certain settings (like ExecStart) it's necessary to clear the old one with this line, otherwise systemd will complain about a duplicate definition.